src/providers/ldap/sdap_async.c

```diff
@@ -80,6 +80,16 @@ static void sdap_call_op_callback(struct sdap_op *op, struct sdap_msg *reply,
               "Handling LDAP operation [%d][%s] took [%"PRIu64"] microseconds.\n",
               op->msgid, op->stat_info == NULL ? "-" : op->stat_info,
               time_spend);
+
+        /* time_spend is in us and timeout in s */
+        if (op->timeout != 0 && (time_spend / op->timeout) >= (80 * 10000)) {
+            DEBUG(SSSDBG_IMPORTANT_INFO, "LDAP operation [%d][%s] seems slow, "
+                                         "took more than 80%% of timeout.\n",
```

Comment:
1) Would it make sense to print `timeout` as well?
2) Would it make sense to send message to syslog? (not sure)

Reply:
I added the `timeout` value. Since the message is shown by default in the SSSD logs I think it is not needed in the syslogs.